dask_cuda/initialize.py

```diff
@@ -44,6 +19,65 @@ def initialize(
     net_devices="",
     cuda_device_index=None,
 ):
+    """Create CUDA context and initialize UCX-Py, depending on user parameters.
+
+    Sometimes it is convenient to initialize the CUDA context, particularly before
+    starting up Dask workers which create a variety of threads.
+
+    To ensure UCX works correctly, it is important to ensure it is initialized with
+    the correct options. This is important for scheduler, workers, and client. This
+    function will ensure that based on the flags and options passed by the user.
+
+    This function can also be used within a Worker preload script.
+    https://docs.dask.org/en/latest/setup/custom-startup.html
+
+    You can add it to your global config with the following YAML:
+
+    .. code-block:: yaml
+
+        distributed:
+          worker:
+            preload:
+              - dask_cuda.initialize
+
+    See https://docs.dask.org/en/latest/configuration.html for more information
+    about Dask configuration.
```

Comment:
We need to add a note here now to let users know that this is already done for `LocalCUDACluster` (including the client, if it's instantiated on the same process as the cluster) and `dask-cuda-worker`, so users don't need to do that again. The common place where users would want to use this is when they instantiate their standalone client connecting to a cluster that has already been started with a combination of `dask-scheduler`/`dask-cuda-worker`s.

Reply:
Okay, in that case I will emphasize in the docstring that this script is meant for use with the client to mirror the pre-configured CUDA cluster/workers, and that the worker preload script would only need to be used with mainline Dask/Distributed.